components/net/image_cache.rs

```diff
@@ -431,6 +436,7 @@ impl ImageCache for ImageCacheImpl {
                 placeholder_url: ServoUrl::parse("chrome://resources/rippy.png").unwrap(),
                 webrender_api: webrender_api,
             })),
+            thread_pool: CoreResourceThreadPool::new(16),
```

Comment:
It put it like that since it was the same as in `components/net/resource_thread.rs`, but here it probably makes more sense to limit it since it is a secondary thread pool. Should I set it to the number of cores then?

Reply:
Yes, perhaps the number of CPUs is a better number. Resource loading is network bound, while decoding is CPU bound.

Reply:
#31585 should address this changes